src/main/java/com/github/minecraftschurlimods/arsmagicalegacy/client/ClientInit.java

```diff
@@ -114,6 +117,8 @@ public static void init() {
     private static void clientSetup(FMLClientSetupEvent event) {
         registerMenuScreens();
         CompatManager.clientInit(event);
+        ItemBlockRenderTypes.setRenderLayer(AMFluids.LIQUID_ESSENCE.get(), RenderType.translucent());
```

Comment:
This shouldn't be necessary anymore? Or do fluids not work with JSON block render types?

Reply:
those are the render types for fluids yes the fluid itself does not have a json model only the fluid block which is not what is being rendered